devicon.json

```diff
@@ -939,11 +957,10 @@
                 "plain-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "plain"
             ]
         },
-        "color": "#008cba",
+        "color": "#3FB6D3",
```

Comment:
I'm fine with changing the color since `3FB6D3` seems more fitting to `foundation`, but why removing the `plain-wordmark` version? It looks fine in my opinion.
Anyway: Changing `foundation` is another topic and should have it's own pull request and review. Would you mind to separate it?

Reply:
Ok, i'll separate flutter `foundation` change on a new pull request.
I removed the `plain-wordmark` as i didn't make `wordmarks`. The only svgs (icons) i've made where original and plain, but none of them are wordmarks.

Reply:
But you removed the `plain-wordmark` version from foundation definition. We have a `plain-wordmark` version of `foundation` https://github.com/devicons/devicon/tree/develop/icons/foundation

Reply:
Sorry, i didn't realized that, i thought another thing 😅 
Let me fix it!